web/controllers/skill_controller.ex

```diff
@@ -0,0 +1,45 @@
+defmodule CodeCorps.SkillController do
+  use CodeCorps.Web, :controller
+
+  alias CodeCorps.Skill
+  alias JaSerializer.Params
+
+  plug :scrub_params, "data" when action in [:create]
+
+  def index(conn, params) do
+    skills =
+      case params do
+        %{"filter" => filter} ->
+          ids = String.split(filter, ",") |> Enum.map(fn(n) -> String.to_integer(n) end)
+          query = Skill |> where([p], p.id in ^ids)
+          Repo.all(query)
+        %{"query" => query} ->
+          text_query = Skill |> where([p], ilike(p.title, ^"#{query}%"))
+          Repo.all(text_query)
```

Comment:
Yeah sounds good. Since this is such a simple indexing it probably wont be hard, but also dont know if its necessary. 
The current query should work well though since responses are now in µs just by being in Elixir? Just need to have an observer on the Ember side text field and it will send a request with every update and return an index with the relevant skills.